tests/utils/path-expressions.js

```diff
@@ -34,3 +34,7 @@ export const getCartItemPathExpression = ( productTitle, args ) =>
 	' and ' +
 	getQtyItemPathExpression( args ) +
 	']';
+
+export const getOrderDetailsDiscountPathExpression = ( amount ) => {
+	return `//tr[th[contains(text(), "Discount:")] and //span[contains(text(), "${ amount }")]]`;
+};
```

Comment:
What do you think about replacing this with a more human friendly check:
```
	await expect(page).toMatchElement(
				`th`,
				{ text: 'Discount' }
			);
       await expect(page).toMatchElement(
				`span.woocommerce-Price-amount`,
				{ text: amount }
			);
```

Reply:
This is indeed a better way to do the check, @ralucaStan! Thank you for this brilliant idea. It will be easier to refactor the code if we decide to change the HTML structure one day. 🙌